### ISSUE TITLE
Preview menu: the link text "Preview in new tab (opens in a new tab)" is redundant

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#24386]
**Describe the bug**

In WordPress 5.5, on large screens the Preview is now a menu with options for desktop, tablet, mobile, and traditional preview in a new browser tab.

The link for the preview in a new browser tab has this text:

"Preview in new tab (opens in a new tab)"

where the portion `(opens in a new tab)` is visually hidden. While it's a recommended practice to add this information to link that open in a new tab and don't inform users about the potentially unexpected new tab opening, in this specific case the link text _already inform users_ as it literally says: "Preview in new tab":

<img width="1045" alt="Screenshot 2020-08-05 at 17 16 57" src="https://user-images.githubusercontent.com/1682452/89442583-5bc13b80-d74f-11ea-9eb3-e136a195379c.png">

Worth also reminding that modern browsers provide tools to check what the computed _accessible name_ (in this case: the link text) is, which makes checking this kind of things very easy:

<img width="1279" alt="Screenshot 2020-08-05 at 17 18 23" src="https://user-images.githubusercontent.com/1682452/89442681-83180880-d74f-11ea-9b53-c4108cba8c94.png">

Above: Chrome dev tools: accessibility tab displaying the computed accessible name of the link.

In this case, the visually hidden text is unnecessary and redundant for screen reader users. It should be removed.

Looking at the implementation, not sure what is the preferred approach. `PostPreviewButton` accepts a `textContent` prop and fallbacks to a link text `Preview` if none is passed. So the default text does need the visually hidden text, while a more meaningful text passed via `textContent` _may_ not need it. Since this depends on the actual text passed via the prop, I'd tend to think the behavior can't be automated and maybe an additional, explicit, prop to render the visually hidden text would be the best option.

**Editor version (please complete the following information):**
- WordPress version: 5.5 RC 2
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? Default.
